### ISSUE TITLE
Shortcut key for imgur upload does not work (click works)

### ISSUE THREAD

**Issue by rtnlmeme-DestroyerOfDeath** [flameshot-org/flameshot#1709]
**Flameshot version**

Flameshot v0.9.0
Compiled with Qt 5.15.2
(pacman package)

**Describe the bug**

The imgur upload key binding does not seem to work.

steps:

optional:
add this line in flamehot.ini file
TYPE_IMAGEUPLOADER=Return

call `flameshot gui` 
make a selection
press "return" or whatever your keybinding for uploading to imgur is

Info:
clicking the imgur upload button works.

**Expected behavior**

Clicking the keybinding for IMGURUPLOADER behaves exactly like clicking the button.

**System Information**
Kernel: 5.10.42-1-MANJARO x86_64 bits: 64 compiler: gcc v: 11.1.0 Desktop: i3 4.19.1 Distro: Manjaro Linux 
           base: Arch Linux 

**Potentially useful info**
It is a bit suspicious that configshortcuts.cpp does not have an entry for our upload in the first part of the code

**Comment by mmahmoudian** [comment]
Would you please perform the following checks:
1. Does other keybindings work?
2. Check if there is no duplicate of the `TYPE_IMAGEUPLOADER` in the condo file
3. If `Return` is set for any other action
3. open the gui config (`flameshot config`) and check what keybinding that window shows for this button

**Comment by rtnlmeme-DestroyerOfDeath** [comment]
Here is my config

```
[General]
contrastOpacity=188
disabledTrayIcon=false
drawColor=#ff00ff
drawThickness=6
savePath=.png
savePathFixed=false
showStartupLaunchMessage=false
startupLaunch=true
uiColor=#00961b

[Shortcuts]
TYPE_ARROW=A
TYPE_CIRCLE=C
TYPE_CIRCLECOUNT=
TYPE_COMMIT_CURRENT_TOOL=
TYPE_COPY=Shift+Y
TYPE_DRAWER=D
TYPE_EXIT=q
TYPE_IMAGEUPLOADER=Return
TYPE_MARKER=M
TYPE_MOVESELECTION=Ctrl+M
TYPE_MOVE_DOWN=J
TYPE_MOVE_LEFT=H
TYPE_MOVE_RIGHT=L
TYPE_MOVE_UP=K
TYPE_OPEN_APP=Ctrl+O
TYPE_PENCIL=p
TYPE_PIN=
TYPE_PIXELATE=b
TYPE_RECTANGLE=r
TYPE_REDO=Ctrl+r
TYPE_RESIZE_DOWN=Shift+J
TYPE_RESIZE_LEFT=Shift+H
TYPE_RESIZE_RIGHT=Shift+L
TYPE_RESIZE_UP=Shift+K
TYPE_SAVE=Shift+S
TYPE_SELECTION=S
TYPE_SELECTIONINDICATOR=
TYPE_SELECT_ALL=Ctrl+A
TYPE_TEXT=i
TYPE_TOGGLE_PANEL=Space
TYPE_UNDO=u
```
1 - yes

2 + 3 -   double checked and I have it with the above config

https://i.imgur.com/XK2uOyr.png

```
flameshot output:

flameshot &
[flameshot] 931020
~/.d/f/.config/flameshot:master* λ QSettings::value: Empty key passed
QSettings::value: Empty key passed
QSettings::setValue: Empty key passed
QSettings::value: Empty key passed
QSettings::setValue: Empty key passed
QPainter::begin: Paint device returned engine == 0, type: 2
QPainter::setRenderHint: Painter must be active to set rendering hints
QPainter::setCompositionMode: Painter not active
QPainter::translate: Painter not active
QPainter::setPen: Painter not active
QPainter::setBrush: Painter not active
QPainter::setBrush: Painter not active
QPainter::begin: Paint device returned engine == 0, type: 2
QPainter::setRenderHint: Painter must be active to set rendering hints
QPainter::setCompositionMode: Painter not active
QPainter::translate: Painter not active
QPainter::setPen: Painter not active
QPainter::setBrush: Painter not active
QPainter::setBrush: Painter not active
```

**Comment by mmahmoudian** [comment]
I replaced my config file with yours and still managed to upload to imgur

Manjaro KDE Xorg + Flameshot v0.10.0 (AUR)

**Comment by rtnlmeme-DestroyerOfDeath** [comment]
Fixed in flameshot 0.10.0
